docs/development.md

```diff
@@ -0,0 +1,21 @@
+# Development
+Additional information and help regarding developing `mgmt`. Useful tools, conventions, etc.
```

Comment:
Maybe this should be a sentence instead? Like "This contains some additional..."

Reply:
Correct.